libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/MapboxOnboardRouter.kt

```diff
@@ -63,7 +64,7 @@ class MapboxOnboardRouter(
                         ""
                     )
                 })
-            navigatorNative.configureRouter(routerParams, null)
+            navigatorNative.configureRouter(routerParams, HttpClient())
```

Comment:
Let's revert this for now. NN if we pass `null` uses the internal client, not sure what would be the effects if we pass an "empty" `HttpClient`. Also, this is not docs 😅 cc @mskurydin

Reply:
👍